packages/ibm-products/src/components/Checklist/Checklist.tsx

```diff
@@ -170,7 +170,7 @@ export let Checklist = React.forwardRef(
       theme = defaults.theme,
       title,
       toggleLabel = defaults.toggleLabel,
-      toggleLabelAlign = defaults.toggleLabelAlign,
+      toggleLabelAlign = 'top',
```

Comment:
We should remove `toggleLabelAlign` from the defaults object if we're not going to use it from there anymore.